samples/apps/copilot-chat-app/webapi/Auth/PassThroughAuthenticationHandler.cs

```diff
@@ -15,6 +16,8 @@ namespace SemanticKernel.Service.Auth;
 public class PassThroughAuthenticationHandler : AuthenticationHandler<AuthenticationSchemeOptions>
 {
     public const string AuthenticationScheme = "PassThrough";
+    private const string DefaultUserId = "c05c61eb-65e4-4223-915a-fe72b0c9ece1";
```

Comment:
There is nothing special about this guid. It just needs to be the same for each request.